### ISSUE TITLE
updating to 3.7.0 something broke 

### ISSUE THREAD

**Issue by Anto79-ops** [timmo001/system-bridge#2568]
### Description

Hi, just updated the newest version on Unbuntu 22.04 and it seems to have broke the bridge. Here what I see

Entities not reporting to Home Assistant anymore (all unavailable, which is different from this [issue](https://github.com/timmo001/system-bridge/issues/2366) ) with this error in the core logs:

![image](https://github.com/timmo001/system-bridge/assets/79593761/19b4521f-02c8-4fd0-9d61-157e851e348c)

```
Logger: homeassistant.config_entries
Source: helpers/aiohttp_client.py:73
First occurred: 8:19:41 AM (1 occurrences)
Last logged: 8:19:41 AM

Error setting up entry Ubuntu (192.168.1.139) for system_bridge
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 390, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/system_bridge/__init__.py", line 71, in async_setup_entry
    if not await version.check_supported():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/systembridgeconnector/version.py", line 40, in check_supported
    and (version := await self.check_version_3()) is not None
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/systembridgeconnector/version.py", line 71, in check_version_3
    response = await self._http_client.get("/api/data/system")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/systembridgeconnector/http_client.py", line 40, in get
    response: ClientResponse = await self.request(
                               ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/systembridgeconnector/http_client.py", line 118, in request
    "response": await response.json(),
                ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/aiohttp_client.py", line 73, in json
    return await super().json(*args, loads=loads, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiohttp/client_reqrep.py", line 1104, in json
    raise ContentTypeError(
aiohttp.client_exceptions.ContentTypeError: 0, message='Attempt to decode JSON with unexpected mimetype: text/plain; charset=utf-8', url=URL('http://192.168.1.139:9170/api/data/system')
```

When using the webinterface, headers are reporting...but its all blank when you expand:

![image](https://github.com/timmo001/system-bridge/assets/79593761/29f8a2d4-5f53-4562-a227-02d35e9061b1)

The service seems to be running normally, however:
![image](https://github.com/timmo001/system-bridge/assets/79593761/a4eb18f0-a6cd-4666-aa4a-f051f0d45c22)

thanks for your help

### What Platform / OS are you running?

Linux

### What version are you running?

3.7.0

**Comment by Anto79-ops** [comment]
Hey @timmo001, any suggestions to get this working? thanks

**Comment by Anto79-ops** [comment]
Hi again,

I updated to 3.8.0 by this command

`python3 -m pip install --upgrade systembridgeshared systembridgebackend systembridgecli systembridgefrontend systembridgegui`

 and it seemed to successfully update but now when I try to run the service

`sudo python3.10 -m systembridgebackend --no-gui`

I get this:

```
anto@anto-m8150n:~$ python3 -m systembridgebackend --no-gui
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 187, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.10/runpy.py", line 146, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.10/runpy.py", line 110, in _get_module_details
    __import__(pkg_name)
  File "/home/antonio/.local/lib/python3.10/site-packages/systembridgebackend/__init__.py", line 15, in <module>
    from .server import Server
  File "/home/antonio/.local/lib/python3.10/site-packages/systembridgebackend/server/__init__.py", line 19, in <module>
    from ..data import Data
  File "/home/antonio/.local/lib/python3.10/site-packages/systembridgebackend/data.py", line 10, in <module>
    from .modules.media import Media
  File "/home/antonio/.local/lib/python3.10/site-packages/systembridgebackend/modules/media/__init__.py", line 10, in <module>
    import winsdk.windows.media.control as wmc  # pylint: disable=import-error
ModuleNotFoundError: No module named 'winsdk'
```

any idea how to solve this one?

**Comment by Anto79-ops** [comment]
thanks @timmo001 both issues described above are now solved on 3.8.1.

Keep you updated on this [issue](https://github.com/timmo001/system-bridge/issues/2366) and see if 3.8.1 solves it.

cheer
